src/styles/index.scss

```diff
@@ -1,3 +1,106 @@
+@import "stars";
+
 body {
+  font-family: $family-roboto;
+  color: #060b35;
   margin: 0;
 }
+
+.card {
+  margin: $margin-center;
+  height: 406px;
```

Comment:
BEM block couldn't have any margins.
Checklist:
7. [BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.
8. [STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.

Reply:
I changed on the h3, because on the page can be some titles with h2. Thanks for the advice